src/app/games/quarto/QuartoTutorial.ts

```diff
@@ -0,0 +1,55 @@
+import { QuartoMove } from 'src/app/games/quarto/QuartoMove';
+import { QuartoPartSlice } from 'src/app/games/quarto/QuartoPartSlice';
+import { QuartoPiece } from 'src/app/games/quarto/QuartoPiece';
+import { DidacticialStep } from '../../components/wrapper-components/didacticial-game-wrapper/DidacticialStep';
+
+export const quartoTutorial: DidacticialStep[] = [
+    DidacticialStep.informational(
+        $localize`Goal of the game`,
+        $localize`Quarto is an alignment game.
+        The goal is to align four pieces that have at least one common aspect:
+        <ul>
+          <li>their color (light or dark)</li>
+          <li>their size (big or small)</li>
+          <li>their pattern (empty or dotted)</li>
+          <li>their shape (round or square)</li>
+        </ul>
+        Here, we have a board with a victory by an alignment of dark pieces.`,
+        new QuartoPartSlice([
+            [14, 12, 4, 2],
+            [16, 16, 16, 16],
+            [16, 16, 16, 16],
+            [16, 16, 16, 16],
+        ], 7, QuartoPiece.ABAB),
+    ),
+    DidacticialStep.anyMove(
+        $localize`Placement`,
+        $localize`Every placement occurs in two steps: placing the piece you have in hand on a square of the board,
+        and picking a piece that the opponent will have to place, by clicking on one of the pieces on the side of the board.
+        If you prefer, the order of these two steps can be reversed.
+        Keep in mind that the second click confirms the move.<br/><br/>
+        Make a move.`,
+        new QuartoPartSlice([
+            [14, 3, 6, 16],
+            [16, 16, 16, 16],
+            [16, 8, 16, 16],
+            [16, 16, 16, 16],
+        ], 7, QuartoPiece.ABAA),
+        new QuartoMove(2, 2, QuartoPiece.BAAB),
+        $localize`Perfect!`,
+    ),
```

Comment:
poopy step, it allow player to give circle piece (and then loose) :| 
(126/152)

Reply:
ticketé ?